components/EditButton.js

```diff
@@ -0,0 +1,7 @@
+export default function EditButton({ onClick }) {
+  return (
+    <button type="button" onClick={onClick}>
+      Edit
+    </button>
+  );
+}
```

Comment:
this component does not add any value. Please use the normal button element instead.